agile-blanche/src/app/components/navbar/Navbar.tsx

```diff
@@ -0,0 +1,94 @@
+import { Disclosure, DisclosureButton, DisclosurePanel} from '@headlessui/react'
+import { Bars3Icon, XMarkIcon } from '@heroicons/react/24/outline'
+import Image from 'next/image';
+
+type navigation = {
+  name: string;
+  href: string;
+  current: boolean;
+};
+
+const navigation = [
```

Comment:
Nit: can specify the type of array this is like this?

```suggestion
const navigation: navigation[] = [
```

Reply:
Yes, no problem.